operators/src/plot/histogram.rs

```diff
@@ -313,6 +323,19 @@ impl HistogramRasterQueryProcessor {
 
         Ok(chart)
     }
+
+    fn empty_histogram(
+        &self,
+    ) -> Result<<HistogramRasterQueryProcessor as PlotQueryProcessor>::OutputFormat> {
+        let histogram =
+            geoengine_datatypes::plots::Histogram::builder(1, 0., 0., self.measurement.clone())
```

Comment:
Thats not working with the ui: 
"Operator: DataTypeError: Plot exception: Histograms max value must be larger than its min value"

Reply:
359ff6d